### ISSUE TITLE
Using a target size (torch.Size([128])) that is different to the input size (torch.Size([128, 1])) is deprecated

### ISSUE THREAD

**Issue by ziqinggao** [HazeDT/DAGCN#3]
您好，请问这个报错怎么处理呢

**Comment by sunjjjppp** [comment]
同求

**Comment by fqy-fqy** [comment]
同求

**Comment by cb123qwe** [comment]
同求，可以加个QQ交流啊484564153

**Comment by leanlivestwan** [comment]
您好，这个解决了吗 这个torch版本是多少啊

**Comment by seedoordog** [comment]
adversarial_label = torch.cat((domain_label_source, domain_label_target), dim=0).view(-1,1).to(self.device)
